packaging/win/deluge-win-installer.nsi

```diff
@@ -105,54 +110,24 @@ Function finishpageaction
     CreateShortCut "$DESKTOP\Deluge.lnk" "$INSTDIR\deluge.exe"
 FunctionEnd
 
-# Test if Visual Studio Redistributables 2008 SP1 installed and returns -1 if none installed
```

Comment:
What is the current status of the redistributables? Will the user always have them installed?